app/src/main/kotlin/pl/gov/mc/protego/ui/base/BaseActivity.kt

```diff
@@ -36,4 +40,18 @@ abstract class BaseActivity : AppCompatActivity() {
         shakeDetector.startDetection(sensorManager)
         shakeDetector.setListener { CockpitMenuLauncher.showCockpit(supportFragmentManager) }
     }
+
+    fun showNoInternetConnectionDialog() {
+        noInternetConnectionDialog =
```

Comment:
Zamiast tego property użyłbym tagu.
Wygląda na to, że w przypadku ponownego utworzenia aktywności np. po zmianie konfiguracji, dialog fragment też się odtworzy automatycznie ale property pozostanie nullem.